scripts/download_webcams.sh

```diff
@@ -0,0 +1,5 @@
+set -eo pipefail
```

Comment:
Let's add a `bash` shebang, in case someone doesn't execute this script with `bash` explicitly. Also, let's `set -u` (fail on unbound variables).

```suggestion
#!/bin/bash
set -euo pipefail
```

Reply:
Done